### PR TITLE
[ TRAFODION 1543 ] and other small changes

### PR DESCRIPTION
**Pull request by amandamoran**

### CODE REVIEW

File: install/installer/traf_config_setup [hunk 1]
```diff
@@ -470,7 +470,7 @@ fi
 #DCS Servers
 
 DCS_SERVERS_PARM=$(($node_count * 8))
-echo -n "Total number of concurrent sessions to start, default [$DCS_SERVERS_PARM]: "
+echo -n "Total number of client concurrent sessions per machine, default [$DCS_SERVERS_PARM]: "
```
Comment (hegdean) [inline install/installer/traf_config_setup]: Do u want to make this "Total number of client connections per node"

Comment (amandamoran) [inline install/installer/traf_config_setup]: Good suggestion. Will do :)

**Comment by amandamoran** [comment]
jenkins, retest

**Comment by svarnau** [comment]
jenkins, retest -- static check fixed by PR143

**Pull request merged**